packages/webcert/src/store/certificate/certificateReducer.ts

```diff
@@ -71,6 +68,8 @@ interface CertificateState {
   routedFromDeletedCertificate: boolean
   functionDisablers: FunctionDisabler[]
   clientValidationErrors: ValidationError[]
+  createdCertificateId: string
+  navigateToCertificate: boolean
```

Comment:
This isn't used and can be removed, right?

Reply:
createdCertificateId is used, but not navigateToCertificate so I will remove that one.